README.md

```diff
@@ -1,113 +1,133 @@
 
-### 2.1 Prerequisite
+### 2.2 Link test case
 
-#### 2.1.1 Setup your aws credentials
+In the PR you create in [AWS Otel Collector](https://github.com/aws-observability/aws-otel-collector) to build the new component,
+[link your testcase](https://github.com/aws-observability/aws-otel-collector/blob/main/e2etest/testcases.json)
 
-please check https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+When you link your testcase, there are five types of testing platforms you can choose,
+* LOCAL, which will run the test in the pr workflow. 
+* EC2, which will run the test in the main branch workflow under an ec2 instance.
+* ECS, which will run the test in the main branch workflow under an ecs cluster.
+* EKS, which will run the test in the main branch workflow under an eks/k8s cluster.
+* SOAKING, which will run every night, and perform high throughput to your test case and monitor the resource usages.
 
-#### 2.1.2 install terraform
+Typically, we required all the types for every test case, if you have a special usage of your component which might not need a certain platform,
+please put the reason in the pr.
+If you find the current test case option can't fulfill your testing requirement, feel free to open an issue, so we can discuss together.
 
-please check https://learn.hashicorp.com/tutorials/terraform/install-cli
+## 3. Run test cases in AWS platform
+
+The commands in Quick Start is to run the test case in containers locally.
+In the case that you want to debug for a certain platform, you can also use this testing framework to run your test case locally in multiple AWS platforms including EC2, ECS and EKS.
+
+### 3.1 Prerequisite
+
+#### 3.1.1 Setup your aws credentials
+please check https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+
+#### 3.1.2 Run Setup
+Setup only needs to be run once, it creates
 
-#### 2.1.3 Run Setup
-Setup only needs to be run once, it creates 
 1. one iam role
 2. one vpc
 3. one security group
 4. one s3 bucket which used to store the ssh key for ec2 login
 
-run
-``
+run 
+````
 cd terraform/setup && terraform init && terraform apply
-``
-
-#### 2.1.4 Build Image
+````
 
+#### 3.1.3 Build Image
 please follow https://github.com/aws-observability/aws-otel-collector/blob/main/docs/developers/build-docker.md to build your image with the new component, and push this image to dockerhub, record the image link, which will be used in your testing.
 
+### 3.2 Run in ECS
 
-### 2.2 Run Test locally
-
-#### 2.2.1 ECS
-
-```shell
+````
 cd terraform/ecs && terraform init && terraform apply -var="aoc_image_repo={{the docker image repo name you just pushed}}" -var="aoc_version={{ the docker image tag name}} -var="testcase=../testcases/{{your test case folder name}}" -var-file="../testcases/{{your test case folder name}}/parameters.tfvars"
```

Comment:
-var="aoc_version={{ the docker image tag name}} -var

missing `"` after `}}`

Reply:
thx